src/components/ScholarshipsMadeSimpleContent.jsx

```diff
@@ -4,70 +4,60 @@ import Typography from '@material-ui/core/Typography';
 import { Link } from 'react-router-dom';
 import Button from '@material-ui/core/Button';
 import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
 import logopic from '../img/blank.png';
 import LoginButton from './LoginButton';
 
-function ContentText(tab) {
-  return (
-    <>
-      <Typography variant="overline" component="h6" gutterBottom>
-        {tab === 'students' ? 'students' : 'community contributor'}
-      </Typography>
-      <Typography variant="h2" component="h2" gutterBottom>
-        {tab === 'students'
-          ? 'Stress-free scholarships'
-          : 'Join and Support the Community'}
-      </Typography>
-      <Typography variant="body2" gutterBottom>
-        Et has minim eltir intellegat. Mea aeterno elefiend antiopam ad, nam no
-        suscipit quaerendum. At name mininum ponderum. Est audiam animal
-        molestiate te.
-      </Typography>
-    </>
-  );
-}
+const useStyles = makeStyles((theme) => ({
+  img: {
+    width: '100%',
+    height: '100%',
+    objectFit: 'contain',
+  },
+}));
 
-function ContentDiplay(user) {
-  if (user === 'students') {
-    return (
-      <>
-        <LoginButton />
-        <Link to="/scholarships">
-          <Button
-            variant="outlined"
-            color="primary"
-            style={{ marginLeft: '20px' }}>
-            Find Scholarships
-          </Button>
-        </Link>
-      </>
-    );
-  }
+function ButtonList(user) {
   return (
     <>
-      <Link to="/home">
-        <Button variant="outlined" color="primary">
-          Submit Scholarship
-        </Button>
-      </Link>
+      {user === 'students' && <LoginButton />}
+      <Button
+        variant="outlined"
+        color="primary"
+        component={Link}
+        style={{ marginLeft: '20px' }}
+        to={user === 'students' ? '/scholarships' : '/scholarships/new'}
+        buttonProperties>
+        {user === 'students' ? 'Find Scholarships' : 'Submit A Scholarship'}
+      </Button>
     </>
   );
 }
 
 export default function ScholarshipsMadeSimpleContent(props) {
+  const classes = useStyles();
   const { user } = props;
```

Comment:
Instead of accepting the user, you could accept properties like `overline`, `title`, `description`, `buttons`, `img` and `direction` (Which gets applied to the `Grid` container so you can put the image first in other sections. See how `direction` works [here](https://material-ui.com/components/grid/#interactive)) and make this a generic component, e.g. `HomeSection`.

This would make the other sections as simple as writing a few lines of code in the `Home` component. I think with a generic `HomeSection` component you could then write https://github.com/beyondhb1079/s4us/pull/163 in about 10 lines or so.

Reply:
Got the generic `HomeSection` implemented and included the test for #163 being displayed in this PR